### ISSUE TITLE
WCAG Audit

### ISSUE THREAD

**Issue by SieuwkeSheta** [Kayleighhhhh/your-tribe-for-life-squad-page#40]
In this issue, the WCAG audit is documented based on a written-out [assignment](https://github.com/fdnd-task/wcag-audit/blob/main/docs/INSTRUCTIONS.md).

***

### Performed tests

- [x] Lighthouse Accessibility test

Manually performed test based on the [de A11Y Project](https://www.a11yproject.com/checklist/) checklist:
- [x] Keyboard
- [x] Screenreader test
- [x] Interactive elements
- [x] Headings and landmarks

**Comment by SieuwkeSheta** [comment]
## Lighthouse Accessibility test
### Homepage
<img width="300" alt="Image" src="https://github.com/user-attachments/assets/4eda51f4-f33c-4aff-b311-9ccd2702c4d8" />

Some bad outcomes:
- Document doesn't have a `<main>` landmark

Some passed audits:
- Background and foreground colors have a sufficient contrast ratio
- Links have a discernible name
- Touch targets have sufficient size and spacing.

### Detail page
<img width="300" alt="Image" src="https://github.com/user-attachments/assets/805a84a9-b0ee-4697-bd5f-ff2128e4357a" />

Some bad outcomes:
- Document doesn't have a `<title>`-element
- Document doesn't have a `<main>` landmark

Some passed audits:
- Definition list items are wrapped in `<dl>` elements
- Touch targets have sufficient size and spacing

## Manual tests
### Keyboard
The tab order follows the logical sequence of the DOM (Document Object Model) or HTML. When you use the Tab key, the focus moves through elements from top to bottom and left to right.

### Screenreader
- The information 'flow' on the page follows a logical order
- The "NVDA" screen reader reads the content from top to bottom and from left to right

### Interactive elements
There is clear visual styling to indicate what a clickable button or link is
- Buttons have different styling in comparison to their surroundings 
  <img height="50" alt="Image" src="https://github.com/user-attachments/assets/258a42e7-b988-48b1-96bb-c1e432dbfca2" /> <img height="50" alt="Image" src="https://github.com/user-attachments/assets/49a72885-739d-4ffc-a0f8-f3be47ac65cc" /> <img height="20" alt="Image" src="https://github.com/user-attachments/assets/cb51e12a-3d28-476f-880e-7e45e3da4a6c" />

- The styling changes when you 'hover' over a clickable button or link

https://github.com/user-attachments/assets/f2eec189-dd27-4001-af5f-6b42f4ceca1a

- The cursor changes to a pointer when you hover over a clickable element

### Headings and landmarks
- Custom styling has been applied to the `<h>`-elements in CSS.
- The `<h>`-elements on the homepage are in the correct order, they are not on the detail page

On the homepage:
<img width="200" alt="Image" src="https://github.com/user-attachments/assets/ae6f1985-c0d5-4507-9bc2-348fb6dd5843" />

On the detail page: 
<img width="200" alt="Image" src="https://github.com/user-attachments/assets/91eb4e29-afa0-405f-a62f-901a0801fe59" />

- Because of the wrong `<h>`-element order on the detail page, the "NVDA" screenreader does not read the `<h>`-elements in the correct order

*** 

Checklist for fix:
- [x] Add `<main>` to documents
- [x] Add `<title>` to detail page
- [ ] Change `<h>` order on detail page
